guides/common/modules/proc_uploading-files-to-a-custom-python-repository.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Uploading_Files_To_a_{custompythontypeid}_Repository_{context}"]
+= Uploading Files To a {custompythontypetitle} Repository
+
+Use this procedure to upload files to a {custompythontype} repository.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select a {customproduct} by name.
+. Select a `python` type repository by name.
+. Click *Browse* to search and select the file you want to upload.
++
+You can upload `.whl` packages and `.tar.gz` archives that contain python packages.
+. Click *Upload* to upload the selected file to {ProjectServer}.
+. Visit the *Published At* URL to see the file.
```

Comment:
This is potentially a verification step.

Reply:
Moved to ".Verification". Is this supposed to be before or after the CLI procedure?

Reply:
After.

Reply:
Moved!